app/ui-react/packages/api/package.json

```diff
@@ -70,4 +80,4 @@
       }
     }
   }
-}
+}
```

Comment:
Why did 'immer' version go back - is it required for monaco?  Same for ts-jest version.  Are the 'patternfly-react' and '@types/patternfly-react' dependencies required here?

Reply:
reverted dependencies and removed patternfly